opendbc/car/ford/radar_interface.py

```diff
@@ -95,13 +95,14 @@ def __init__(self, CP):
 
     self.points: list[list[float]] = []
     self.clusters: list[Cluster] = []
+    self.rcp = None
+    if CP.radarUnavailable:
+      return
 
     self.updated_messages = set()
     self.track_id = 0
-    self.radar = DBC[CP.carFingerprint]['radar']
-    if CP.radarUnavailable:
-      self.rcp = None
-    elif self.radar == RADAR.DELPHI_ESR:
+    self.radar = DBC[CP.carFingerprint][Bus.RADAR]
```

Comment:
```suggestion
    self.radar = DBC[CP.carFingerprint].get(Bus.RADAR)
```

can also just do this